landoapi/hg.py

```diff
@@ -129,18 +133,27 @@ def _config_to_list(self):
         return ["{}={}".format(k, v) for k, v in self.config.items() if v is not None]
 
     def __enter__(self):
+        if self.requester_email:
+            # Set the environment variable the `set_landing_system.py` hg
```

Comment:
There might be a word or punctuation missing here.

Reply:
This might be a remnant of something I copy and pasted, nice catch, will change.

Reply:
Ended up just adjusting the line width here, I think the sentence makes sense but kind of awkward to read.